arroyo-rpc/proto/api.proto

```diff
@@ -581,6 +581,9 @@ message SourceFieldType {
     PrimitiveType primitive = 1;
     StructType struct = 2;
   }
+
+  // include a sql name if the type is PrimitiveType
```

Comment:
Rather than showing `""`, maybe make it optional?

Reply:
Sure, done.